lib/semantics/check-do-concurrent.cc

```diff
@@ -12,11 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "check-do-concurrent.h"
 #include "attr.h"
+#include "check-do-concurrent.h"
```

Comment:
LLVM coding rules say that the header for the module must come first.  Newer clang-format versions get this right.  Older versions do not and create diff entropy.

Reply:
git-clang-format is my new buddy.